wiki/News_Styling_Criteria/en.md

```diff
@@ -112,7 +116,9 @@ Images used in news articles must be in JPG format at quality 9 (90) to quality
 
 #### File size
 
-All images must be optimized as much as possible. We recommend the use of [FileOptimizer](http://nikkhokkho.sourceforge.net/static.php?page=FileOptimizer) to do this quickly and consistently.
+All images must be optimised as much as possible. We recommend the use of [FileOptimizer](http://nikkhokkho.sourceforge.net/static.php?page=FileOptimizer) to do this quickly and consistently.
```

Comment:
might be worth it to go more in-depth here and mention [jpeg-archive](https://github.com/danielgtaylor/jpeg-archive)'s jpeg-recompress as well, this is what Ephemeral recommended to me

Reply:
Hmm, alright. I'll replace FileOptimizer with jpeg-archive. An update from FileOptimizer added adverts.

Reply:
reminder if I forget:
![image](https://user-images.githubusercontent.com/14264992/40953217-628a17c4-6844-11e8-8276-ae075dd033b3.png)